source/reference/system-environment-variables.txt

```diff
@@ -0,0 +1,18 @@
+.. _system-environment-variables:
+
+----------------------------
+System Environment Variables
```

Comment:
I think we should call them either system variables or environment variables, but not both.